### ISSUE TITLE
sdist is missing tox.ini

### ISSUE THREAD

**Issue by mtelka** [python-hyper/h11#164]
The sdist package at PyPI is missing `tox.ini`.  Please add the missing `tox.ini` file to sdist to make downstream testing easier.  Thank you.

**Comment by mtelka** [comment]
I'm packaging `h11` (and hundreds of other Python projects) for OpenIndiana and tests included in sdist are common.  Yes, there are some people that do not want tests in sdist but they are minority only.

An recent example: https://github.com/pypa/distlib/issues/206

I have plenty of similar cases when some files needed for tests were missing and after a bug report such files were added.
There is also (much bigger) list of projects that include full tests properly since day zero.

Thank you.

**Comment by zanieb** [comment]
Makes sense. The tests are already in the main module here so it seems reasonable. Perhaps you want to open a pull request? It's trivial to add e.g. https://github.com/pypa/distlib/commit/ff48e09e9b5f8588145e22f5b7c51b361eb1fcfe

I'm not a maintainer here though — just a contributor.
